packages/admin/cms-admin/src/redirects/RedirectsTable.tsx

```diff
@@ -284,3 +286,8 @@ const IconWrapper = styled("div")`
     display: flex;
     flex-direction: row;
 `;
+
+const DataGridContainer = styled("div")`
+    height: calc(100vh - 200px);
```

Comment:
Does MUI provide a helper to adjust the size of the DataGrid to fit the available space?

Reply:
So far I have not been able to find anything else that would allow the grid to take on the full height. There is an autoHeight prop for the data grid, but it refers to the rows and thus the grid itself would not fill the entire space. 

<img width="1728" alt="Screenshot 2022-12-20 at 06 59 32" src="https://user-images.githubusercontent.com/24371358/208594834-dfdbd35f-fc7a-4a78-9bc0-13d693129d70.png">

Reply:
We may come up with a general purpose solution in the future.

Reply:
Here I have now temporarily used the same styling as for the Products Table.